TinyKeychain/KeychainKey.swift

```diff
@@ -36,7 +36,7 @@ public extension Keychain {
      ```
      
      */
-    public struct Key<Object: Codable> {
+    struct Key<Object: Codable> {
```

Comment:
Not necessary, but it's redundant since the extension itself is marked as `public`. Xcode gives you a warning when trying to compile using Swift 5.

`'public' modifier is redundant for generic struct declared in a public extension`